electron/logger.ts

```diff
@@ -0,0 +1,27 @@
+import { createLogger, format, transports }from 'winston'
+
+const Logger = createLogger({
+  //defaultMeta: { service: 'heroic-main' },
+  exceptionHandlers: [
+    new transports.File({ filename: 'logs/heroic-issues.log' })
```

Comment:
Would prefer if this was changed to heroic-error.log. Also do we really need the `heroic-` prefix?

Reply:
Was thinking about piping the output from wine or legendary to a log in the same dir so would've needed clarification for that.

Reply:
Oh alright. Please rename the exception handler to `heroic-crashes` and `heroic-issues` to `heroic-errors` though.

Reply:
Maybe each element should have its own folder inside of logs, ex, "/logs/heroic;/logs/wine;/logs/legendary;"